deepface/DeepFace.py

```diff
@@ -396,7 +396,7 @@ def represent(
             (default is True).
 
```

Comment:
you added some facial recognition models for yolo but they are missing here

Reply:
fixed

Reply:
there is no Yolo-Face in modelling